### PR TITLE
Remove textToDisplay in favor of inline

### PR DESCRIPTION
**Pull request by Dinnall**
Remove textToDisplay in favor of inline
add toSting() to `campaignId` and  `userId`

### What's this PR do?

This pull request fixes an error message when viewing the scholarship page
graphql requires both `campaignId` and  `userId`, added toString() to fix the error

Also switched to an inline code for the `text` prop and checked if the data is available before evaluating what to be presented 
### How should this be reviewed?
Screen Shot 2020-12-01 at 12 42 20 PM

Screen Shot 2020-12-01 at 5 55 17 PM

...

### Any background context you want to provide?

...

### Relevant tickets

References [Pivotal #173454908](https://www.pivotaltracker.com/n/projects/2401401/stories/173454908/comments/220130129).

### Checklist

- [ x ] This PR has been added to the relevant Pivotal card.
- [ x  ] Added screenshots of front-end changes on small, medium, and large screens.

### CODE REVIEW

File: resources/assets/components/utilities/ScholarshipCard/GalleryBlockSignup.js [hunk 1]
```diff
@@ -37,8 +37,8 @@ const GalleryBlockSignup = ({ campaignId, path }) => {
     error: errorCampaign,
   } = useQuery(SEARCH_USER_CAMPAIGN_QUERY, {
     variables: {
-      userId: window.AUTH.id,
-      campaignId,
+      userId: window.AUTH.id.toString(),
```
Comment (aaronschachter) [inline resources/assets/components/utilities/ScholarshipCard/GalleryBlockSignup.js:40]: Can we DRY this by using the existing [helper](https://github.com/DoSomething/phoenix-next/blob/75e9a3098d5057ba8eb69dadd991d4ec330225ed/resources/assets/helpers/auth.js#L88)?

Comment (Dinnall) [reply resources/assets/components/utilities/ScholarshipCard/GalleryBlockSignup.js]: Forgot about that helper. Helper imported, done, and done 👍

File: resources/assets/components/utilities/ScholarshipCard/GalleryBlockSignup.js [hunk 2]
```diff
@@ -37,8 +37,8 @@ const GalleryBlockSignup = ({ campaignId, path }) => {
     error: errorCampaign,
   } = useQuery(SEARCH_USER_CAMPAIGN_QUERY, {
     variables: {
-      userId: window.AUTH.id,
-      campaignId,
+      userId: getUserId,
```
Comment (DFurnes) [inline resources/assets/components/utilities/ScholarshipCard/GalleryBlockSignup.js:40]:
```suggestion
      userId: getUserId(),
```

Missing parens!

**Review by DFurnes — APPROVED** [review]
Looks good!

**Pull request merged**